kitsune/search/v2/search.py

```diff
@@ -114,7 +115,13 @@ def get_highlight_fields(self):
         ]
 
     def get_filter(self):
-        filters = [DSLQ("term", _index=self.get_index())]
+        # Add default filters:
+        # 1. limit scope to the Wiki index
+        # 2. exclude archived articles
+        filters = [
+            DSLQ("term", _index=self.get_index()),
+            DSLQ("term", is_archived=False),
```

Comment:
`is_archived` is a multi-locale field, so as written this will filter out all wiki documents. The field name needs to be `f"is_archived.{self.locale}"`.